app/src/main/java/com/kickstarter/ui/activities/compose/login/KSLoginDropdownMenu.kt

```diff
@@ -27,62 +27,68 @@ fun KSLoginDropdownMenu(
     onPrivacyPolicyClicked: () -> Unit,
     onCookiePolicyClicked: () -> Unit,
     onHelpClicked: () -> Unit
-
 ) {
     DropdownMenu(
-        modifier = Modifier
-            .background(color = KSTheme.colors.kds_support_100)
-            .testTag(LoginDropdownTestTag.OPTIONS_MENU.name),
         expanded = expanded,
-        onDismissRequest = { onDismissed.invoke() }
+        onDismissRequest = { onDismissed.invoke() },
+        modifier = Modifier.testTag(LoginDropdownTestTag.OPTIONS_MENU.name),
+        containerColor = KSTheme.colors.kds_support_100
     ) {
+        val itemColors = MenuDefaults.itemColors(
+            textColor = KSTheme.colors.kds_support_700
+        )
+
         DropdownMenuItem(
             onClick = {
                 onTermsOfUseClicked.invoke()
                 onDismissed.invoke()
+            },
+            colors = itemColors,
+            text = {
+                Text(
+                    modifier = Modifier.testTag(LoginDropdownTestTag.OPTIONS_TERMS.name),
+                    text = stringResource(R.string.login_tout_help_sheet_terms)
+                )
             }
-        ) {
-            Text(
-                modifier = Modifier.testTag(LoginDropdownTestTag.OPTIONS_TERMS.name),
-                text = stringResource(id = R.string.login_tout_help_sheet_terms),
-                color = KSTheme.colors.kds_support_700
-            )
-        }
+        )
         DropdownMenuItem(
             onClick = {
                 onPrivacyPolicyClicked.invoke()
                 onDismissed.invoke()
+            },
```

Comment:
The `text` is now within the `DropdownMenuItem`, colors applied to `DropdownMenuItem` instead to the `text`